src/timeline/Timeline.js

```diff
@@ -0,0 +1,216 @@
+// @flow
+import {
+  View,
+  Text,
+  ScrollView,
+  TouchableOpacity,
+  Dimensions,
+} from 'react-native';
+import populateEvents from './Packer';
+import React from 'react';
+import moment from 'moment';
+import _ from 'lodash';
+import styleConstructor from './style';
+
+const LEFT_MARGIN = 60 - 1;
+// const RIGHT_MARGIN = 10
+// const CALENDER_HEIGHT = 2400;
+// const EVENT_TITLE_HEIGHT = 15
+const TEXT_LINE_HEIGHT = 17;
+// const MIN_EVENT_TITLE_WIDTH = 20
+// const EVENT_PADDING_LEFT = 4
+
+function range(from, to) {
+  return Array.from(Array(to), (_, i) => from + i);
+}
+
+let {width: dimensionWidth} = Dimensions.get('window');
+
+export default class Timeline extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    const {start = 0, end = 24} = this.props;
```

Comment:
Why not use `defaultProps` object instead of setting the default values every time?

Reply:
@Inbal-Tish It was just only two values, so i neglected default props. However I have added defaultProps now, to be on conventions.